### PR TITLE
Update upgrade.sh to support arm ocp

### PR DESCRIPTION
**Pull request by liqcui**
Update script to support arm ocp environment

### CODE REVIEW

**Comment by liqcui** [comment]
/cc @paigerube14 @qiliRedHat 
Please review my PR for support arm ocp upgrade, thanks!

File: upgrade_scripts/upgrade.sh [hunk 1]
```diff
@@ -262,9 +279,9 @@ do
         fi
     else
       if [ "X$enable_force" == "Xtrue" ];then
-        upgrade_line="oc adm upgrade --to-image quay.io/openshift-release-dev/ocp-release:$target_version_prefix-x86_64 --force --allow-explicit-upgrade"
+        upgrade_line="oc adm upgrade --to-image quay.io/openshift-release-dev/ocp-release:${target_version_prefix}-${arch_prefix} --force --allow-explicit-upgrade"
       else
-        upgrade_line="oc adm upgrade --to-image quay.io/openshift-release-dev/ocp-release:$target_version_prefix-x86_64 --allow-explicit-upgrade"
+        upgrade_line="oc adm upgrade --to-image quay.io/openshift-release-dev/ocp-release:${target_version_prefix}-{arch_prefix} --allow-explicit-upgrade"
```
Comment (paigerube14) [inline upgrade_scripts/upgrade.sh:284]: Think you're missing a "$" here 

`upgrade_line="oc adm upgrade --to-image quay.io/openshift-release-dev/ocp-release:${target_version_prefix}-${arch_prefix} --allow-explicit-upgrade"`

File: upgrade_scripts/upgrade.sh [hunk 2]
```diff
@@ -216,20 +216,37 @@ function check_upgrade_version() {
...
       if [[ "X$eus" == "Xtrue" ]]; then
         python3 -c "import check_upgrade; check_upgrade.pause_machinepool_worker('true')"
-        target_sha=$(python3 -c "import check_upgrade; check_upgrade.get_sha_url('https://amd64.ocp.releases.ci.openshift.org/graph','$target_version_prefix')")
+        #Check if the worker node is arm64 or x86_64
+        if [[ "X$node_arch" == "Xarm64" ]];then
+          target_sha=$(python3 -c "import check_upgrade; check_upgrade.get_sha_url('https://arm64.ocp.releases.ci.openshift.org/graph','$target_version_prefix')")
+        else
+          target_sha=$(python3 -c "import check_upgrade; check_upgrade.get_sha_url('https://amd64.ocp.releases.ci.openshift.org/graph','$target_version_prefix')")
+        fi
         if [[ $target_sha == "" ]]; then
```
Comment (paigerube14) [inline upgrade_scripts/upgrade.sh:248]: Could we add in this if statement into each of the target_sha setting lines above? This way we are able to properly print off the url that the version could not be found

File: upgrade_scripts/upgrade.sh (hunk 1 above)
Comment (liqcui) [reply upgrade_scripts/upgrade.sh]: Fixed in new code, thank you

File: upgrade_scripts/upgrade.sh (hunk 2 above)
Comment (liqcui) [reply upgrade_scripts/upgrade.sh]: Fixed in new code, thank you for your suggestion

**Comment by liqcui** [comment]
@qiliRedHat @paigerube14, re-run the loaded upgrade job using my new upgrade.sh, the job log is https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/liqcui-e2e-benchmarking-multibranch-pipeline/job/upgrade/28/console

**Comment by liqcui** [comment]
@qiliRedHat @paigerube14, re-run the loaded upgrade job using my new upgrade.sh, the job log is https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/liqcui-e2e-benchmarking-multibranch-pipeline/job/upgrade/28/console

**Pull request merged**